erts/emulator/beam/dist.c

```diff
@@ -2516,6 +2542,31 @@ int erts_net_message(Port *prt,
 #endif
         from = tuple[2];
         to = tuple[3];
+        if (is_list(to)) {
+            Eterm pid;
+            if (context) {
+                // If we're passing a context, save the state and trap,
```

Comment:
I wanted to test that this would work with a list of a single pid while ensuring the whole yielding mechanism is correct, so I implemented it this way initially. A good question is how many sends to do before yielding too 🤔